pry-sorbet.gemspec

```diff
@@ -25,6 +25,9 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
+  spec.add_dependency "pry"
+
   spec.add_development_dependency "bundler", "~> 2.0"
   spec.add_development_dependency "rake", "~> 10.0"
+  spec.add_development_dependency "sorbet-runtime"
```

Comment:
Would adding a version constraint to these new dependencies be a good idea, or is that unnecessary for these?

Reply:
For a gem, version constraints do not make too much sense, unless there is reason to believe that the gem is not compatible with some specific range of versions. So it is best to use `>= xxx` than `~` or to just not specify any version constraints at all. After all, we don't want to prevent people from trying this gem with newer versions of `sorbet-runtime` or `pry`. It might break and they will report it, but if we restrict then they can't even test it.

For this case, specifically, I guess we should add a `>= 0.5` for `sorbet-runtime` since it is probably not compatible with `0.4` series anymore.

Reply:
I am not sure about the `pry` case, though. I'd like to err on the side of being open and add constraints if people complain that `pry-sorbet` is not working with a particular version of `pry`.

Reply:
Btw, the previous code was implicitly depending on `pry` at runtime, so was basically not specifying a constraint on `pry` version either. So this does not change things.

Reply:
Alright, that's fine then!